CHANGELOG.md

```diff
@@ -17,12 +17,25 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased](https://github.com/o1-labs/snarkyjs/compare/c5a36207...HEAD)
 
+> There are no unreleased changes yet
+
+## [0.9.0](https://github.com/o1-labs/snarkyjs/compare/c5a36207...HEAD)
+
 ### Added
 
 - `Transaction.fromJSON` to recover transaction object from JSON https://github.com/o1-labs/snarkyjs/pull/705
 - New precondition: `provedState`, a boolean which is true if the entire on-chain state of this account was last modified by a proof https://github.com/o1-labs/snarkyjs/pull/741
   - Same API as all preconditions: `this.account.provedState.assertEquals(Bool(true))`
   - Can be used to assert that the state wasn't tampered with by the zkApp developer using non-contract logic, for example, before deploying the zkApp
+- New on-chain value `globalSlot`, to assert that a method can only in a certain time range https://github.com/o1-labs/snarkyjs/pull/649
```

Comment:
... can only _what_ in a certain time range? Perform account updates? I think there's a word or two missing :)